linter.py

```diff
@@ -2,44 +2,23 @@
 # linter.py
 # Linter for SublimeLinter3, a code checking framework for Sublime Text 3
 #
-# Written by Luc Bettaieb, code modefied from NotSqrt
+# By Carlos Asmat. Derived from Luc Bettaieb's work
```

Comment:
Mind replacing this with 
Authors: Luc Bettaieb, Carlos Asmat
?

Reply:
Sure, I'll change it.